packages/devextreme/js/__internal/ui/chat/chat.ts

```diff
@@ -3,14 +3,26 @@ import $ from '@js/core/renderer';
 import type { Properties } from '@js/ui/chat';
 
 import Widget from '../widget';
+import ChatHeader from './chat_header';
+import MessageBox from './chat_message_box';
+import MessageList from './chat_message_list';
 
 const CHAT_CLASS = 'dx-chat';
 
+const MOCK_CURRENT_USER_ID = 'CURRENT_USER_ID';
+
 class Chat extends Widget<Properties> {
+  _chatHeader?: ChatHeader;
+
+  _messageBox?: MessageBox;
+
+  _messageList?: MessageList;
+
   _getDefaultOptions(): Properties {
     return {
       ...super._getDefaultOptions(),
       ...{
+        title: '',
```

Comment:
Does we have test for checking default value of title? (e.g. defaultOptions.tests.js)

Reply:
[90dd3acd3c6dd8b06c21608a237d774ee8b6aa43]